content/en/docs/tasks/observability/metrics/tcp-metrics/index.md

```diff
@@ -0,0 +1,161 @@
+
+        {{< text bash >}}
+        $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all.yaml@
+        {{< /text >}}
+
+        If you enabled mutual TLS, please run the following instead
+
+        {{< text bash >}}
+        $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all-mtls.yaml@
+        {{< /text >}}
+
+        You can display the destination rules with the following command:
+
+        {{< text bash >}}
+        $ kubectl get destinationrules -o yaml
+        {{< /text >}}
+
+        Since the subset references in virtual services rely on the destination rules,
+        wait a few seconds for destination rules to propagate before adding virtual services that refer to these subsets.
+
+    1.  Create `ratings` and `reviews` virtual services:
+
+        {{< text bash >}}
+        $ kubectl apply -f @samples/bookinfo/networking/virtual-service-ratings-db.yaml@
+        Created config virtual-service/default/reviews at revision 3003
+        Created config virtual-service/default/ratings at revision 3004
+        {{< /text >}}
+
+1.  Send traffic to the sample application.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+1.  Verify that the TCP metric values are being generated and collected.
+
+    In a Kubernetes environment, setup port-forwarding for Prometheus by
+    executing the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 &
+    {{< /text >}}
+
+    View values for the TCP metrics in the Prometheus browser window.  Select **Graph**.
+    Enter the `istio_tcp_connections_opened_total` metric or `istio_tcp_connections_closed_total` and select **Execute**.
+    The table displayed in the
+    **Console** tab includes entries similar to:
+
+    {{< text plain >}}
+    istio_tcp_connections_opened_total{destination_version="v1",instance="172.17.0.18:42422",job="istio-mesh",source_service="ratings-v2",source_version="v2"}
+    {{< /text >}}
+
+    {{< text plain >}}
+    istio_tcp_connections_closed_total{destination_version="v1",instance="172.17.0.18:42422",job="istio-mesh",source_service="ratings-v2",source_version="v2"}
+    {{< /text >}}
+
+## Understanding TCP telemetry collection
+
+In this task, you used Istio configuration to
+automatically generate and report metrics for all traffic to a TCP service
+within the mesh.
+
+### TCP attributes
+
+Several TCP-specific attributes enable TCP policy and control within Istio.
+These attributes are generated by Envoy Proxies and obtained from Istio using Envoy's Node Metadata.
+Peer Envoys are forwarded Node Metadata based on ALPN based tunneling with a prefixed based protocol.
```

Comment:
Is there a way to rewrite this sentence to be more clear? It's hard to parse. Something more like "X forwards Node Metadata to Peer Envoys using ALPN based tunneling and a prefixed protocol" or similar?

Reply:
How about?
Envoy forwards Node Metadata to Peer Envoys using ALPN based tunneling and a prefixed protocol

Reply:
Agree the original sentence is confusion. This sentence is better. Should it be `prefix based` or `prefixed based` protocol (no _based_ in the later sentence)?

Reply:
Thanks, corrected and yes it should be prefix based